src/doghouse/adapters/git/git_adapter.py

```
@@ -0,0 +1,40 @@
+from typing import List
+from ...core.domain.blocker import Blocker, BlockerType, BlockerSeverity
+
+class GitAdapter:
+    """Adapter for local git repository operations."""
+    
+    def get_local_blockers(self) -> List[Blocker]:
+        """Detect local issues (uncommitted, unpushed)."""
+        blockers = []
+        
+        # Check for uncommitted changes
+        status = subprocess.run(["git", "status", "--porcelain"], capture_output=True, text=True).stdout
```

Ruff (0.15.7)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)